app/controllers/spree/admin/base_controller.rb

```diff
@@ -81,6 +84,12 @@ def check_json_authenticity
 
       private
 
+      def page_load_request?
+        return false if request.format.include?('turbo')
+
+        html_request?
+      end
```

Comment:
I vaguely remember talking about this before.. I guess we didn't merge it last time?

Reply:
Yes, I took it from here: https://github.com/openfoodfoundation/openfoodnetwork/pull/12672#pullrequestreview-2208870777 😅